src/webui/www/private/scripts/client.js

```diff
@@ -358,14 +348,11 @@ window.addEvent('load', function() {
             return false;
 
         let removed = false;
-        for (const key in category_list) {
-            if (!Object.hasOwn(category_list, key))
-                continue;
-
-            const category = category_list[key];
+        category_list.forEach((category) => {
```

Comment:
Maybe also fix `category_list` -> `categoryList`

Reply:
I've tried it but there were name clashes with local variables, so maybe later.